docs/content/mongocxx-v3/installation.md

```diff
@@ -118,92 +118,128 @@ obtain.
 
 ### Step 4: Configure the driver
 
-On Unix systems, `libmongoc` installs into `/usr/local` by default.  To
-configure `mongocxx` for installation into `/usr/local` as well, use the
-following `cmake` command:
-
-(***NOTE***: The trailing `..` below is important!  Don't omit it.)
+On Unix systems, `libmongoc` installs into `/usr/local` by default. Without additional
+configuration, `mongocxx` installs into its local build directory as a courtesy to those who build
+from source. To configure `mongocxx` for installation into `/usr/local` as well, use the following
+`cmake` command:
 
 ```
-cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local ..
+cmake ..                                \
+    -DCMAKE_BUILD_TYPE=Release          \
+    -DCMAKE_INSTALL_PREFIX=/usr/local
 ```
 
-`/usr/local` may be replaced with the directory that `libmongoc` was
-installed to.
+> ***NOTE*** The trailing `..` above is important!  Don't omit it.
```

Comment:
this is now a leading `..` 😸

Reply:
Well, it trails its line. The entire note is somewhat patronizing, you won't be able to build without some understanding of CMake.